js/app.js

```diff
@@ -0,0 +1,25 @@
+window.addEventListener("scroll", function () {
+  const fixedHeader = document.querySelector(".header");
+  const topbar = document.querySelector(".topbar");
+  const menubar = document.querySelector(".menubar");
+
+  fixedHeader.classList.toggle("fixed", window.scrollY > 0);
+  topbar.classList.toggle("display-none", window.scrollY > 0);
+  menubar.classList.toggle("menubar-white", window.scrollY > 0);
+});
+
+const searchIcon = document.querySelector('.fa-search');

```

Comment:
jsで使われていることを明示した方がいいですがスタイルも当てていますか？
```suggestion
const searchIcon = document.querySelector('.js-fa-search');

```

あとfaとはなんですか？

Reply:
faは Font Awesome のクラス名です。
新たに js-fa-search を追加したいと思います。

Reply:
> faは Font Awesome のクラス名です。

仕様でそうなっているということですか？

Reply:
そうです。
>You can place Font Awesome icons just about anywhere using the CSS Prefix fa and the icon's name.
https://fontawesome.com/v4.7.0/examples/